src/CookieActionComponent.cpp

```diff
@@ -10,24 +12,16 @@
 CookieActionComponent::CookieActionComponent() {
 
     aimInputComponent = new AimInputComponent();
-    cookieShape = std::make_shared<Shape>();
-    cookieShape->loadMesh(RESOURCE_DIR + "sphere.obj");
-    cookieShape->resize();
-    cookieShape->init();
-
-    static std::shared_ptr<Shape> shape = std::make_shared<Shape>();
-    static bool hasLoaded = false;
-
-    if (!hasLoaded) {
-        shape->loadMesh(RESOURCE_DIR + "sphere.obj");
-        shape->resize();
-        shape->init();
-        hasLoaded = true;
-    }
 
-    BunnyRenderComponent* bunnyRenderComp = new BunnyRenderComponent(shape, "Phong", brass);
-    BunnyRenderComponent* bunnyRenderComp1 = new BunnyRenderComponent(shape, "Phong", brass);
-    BunnyRenderComponent* bunnyRenderComp2 = new BunnyRenderComponent(shape, "Phong", brass);
+    ShapeManager& shapeManager = ShapeManager::instance();
+    MaterialManager& materialManager = MaterialManager::instance();
+
+    BunnyRenderComponent* bunnyRenderComp = new BunnyRenderComponent(
```

Comment:
I want to eventually add this to the JSON file as like a directional component or something, for now this is fine though.